src/net/parsers/texttracks/webvtt.js

```diff
@@ -0,0 +1,311 @@
+/** 
+* Parse WebVTT from text. Returns an array with:
+* - start : start of current cue, in seconds
+* - end : end of current cue, in seconds
+* - content : HTML formatted cue.
+*
+* Global style is parsed and applied to div element. 
+* Specific style is parsed and applied to class element. 
+*
+* @param {string} text
+* @return {Array<Object>}
+*/
+export default function parseWebVTT(text) {
+  const newLineChar = /\n|\s{2,}/g;
+  const linified = text.split(newLineChar);
+  const cuesArray = [];
+  const styleElements = [];
+
+  if (!linified[0].match(/^WEBVTT.*/)) {
```

Comment:
The spec says WEBVTT should either be followed by a space, a tab or a line terminator.

But we are more tolerant here, so why not.